datadog/tests/resource_datadog_powerpack_iframe_test.go

```diff
@@ -0,0 +1,41 @@
+package test
+
+import (
+	"testing"
+)
+
+const datadogPowerpackIFrameTest = `
+resource "datadog_powerpack" "iframe_powerpack" {
+	name         = "{{uniq}}"
+    tags = ["tag:foo1"]
+	description   = "Created using the Datadog provider in Terraform"
+	template_variables {
+		defaults = ["defaults"]
+		name     = "datacenter"
+	}
+    widget {
+      iframe_definition {
+        url = "https://google.com"
+      }
+    }
```

Comment:
```suggestion
    name         = "{{uniq}}"
    tags = ["tag:foo1"]
    description   = "Created using the Datadog provider in Terraform"
    template_variables {
        defaults = ["defaults"]
        name     = "datacenter"
    }
    widget {
        iframe_definition {
            url = "https://google.com"
        }
    }
```

Nit. Could we format the resources everywhere it looks like it happens in many of the new tests and would be nice